FirebaseAdmin/FirebaseAdmin/Auth/FirebaseToken.cs

```diff
@@ -80,9 +87,20 @@ internal FirebaseToken(Args args)
         /// </summary>
         public IReadOnlyDictionary<string, object> Claims { get; }
 
+        /// <summary>
+        /// Defined operator string.
+        /// </summary>
+        /// <param name="v">FirebaseToken.</param>
+        public static implicit operator string(FirebaseToken v)
+        {
+            throw new NotImplementedException();
```

Comment:
Please implement this.

Reply:
I have updated the auth/FirebaseToken.cs file, and the issue has been resolved.